include/msgpack/v1/vrefbuffer.hpp

```diff
@@ -26,11 +26,16 @@
 
 #if defined(unix) || defined(__unix) || defined(__APPLE__) || defined(__OpenBSD__)
 #include <sys/uio.h>
+namespace msgpack {
+typedef ::iovec iovec;
+} // namespace msgpack
```

Comment:
I think that this modification has already been merged.
I guess that rebase process is not done well.